driver-core/src/main/com/mongodb/internal/operation/FindOperation.java

```diff
@@ -620,6 +621,32 @@ public boolean getRetryReads() {
         return retryReads;
     }
 
+    /**
+     * Returns the allowDiskUse value
+     *
+     * @return the allowDiskUse value
+     * @since 4.0
```

Comment:
Same as above.  This class is internal so no need for `@since`.

Reply:
Done